TestApp/app/App.js

```diff
@@ -3,7 +3,6 @@
 
 import { AppState, Alert, Platform, ToastAndroid } from 'react-native';
 import { createBottomTabNavigator, createAppContainer } from 'react-navigation';
-import Toast from 'react-native-simple-toast';
```

Comment:
`react-native-simple-toast` npm package is not compatible with iOS autolinking and I am not sure if there's any workaround.

We use `react-native-simple-toast` in TestApp to send us toast message to improve the usability of the app. Without toast, we will need to check the XCode console logs or Android Studio Logcat.

Let's remove it for now and create an item in the backlog to fix it later.